docs/architecture.md

```diff
@@ -1,3 +1,11 @@
 # DSCP Architecture
 
+DSCP architecture has two variations.
+
+- `v1` interacts with `dscp-node` and `dscp-ipfs` using `dscp-api`.
+- `v2` interacts with `dscp-node` and `dscp-ipfs` within the external API service e.g. `dscp-matchmaker-api`.
```

Comment:
I wouldn't call these v1 and v2.  Using the `dscp-api` is a use-case choice.  We are using the `dscp-matchmaker-service` for a specific use-case which requires us to monitor onchain state.

Reply:
changed to use case one/two, what do you think?